src/components/CardModal/CardModalActions/TagsMenu/TagOption/TagOption.js

```diff
@@ -17,8 +17,8 @@ const TagOption = ({ tag, isSelected, editTag, tagClicked }) => {
   };
 
   return (
-    <div onClick={handleClick} className="flex h-8">
-      <div ref={tagRef} className="items-center flex text-white w-full rounded-ibsm cursor-pointer">
+    <div className="flex h-8">
+      <div ref={tagRef} className="items-center flex text-white w-full rounded-ibsm cursor-pointer" onClick={handleClick}>
```

Comment:
Small bug fix here. Previously, if you selected the edit tag pencil, it would toggle the tag on the card i.e. if it was selected, it would be unselected. This is because the onClick was on the parent div, when it should've been on the tag child div